### ISSUE TITLE
Payload values should override builder provided payload props

### ISSUE THREAD

**Issue by jamischarles** [krakenjs/beaver-logger#71]
I would expect payload values passed in directly to logger methods to override builder provided props.

The opposite is true. I think we should consider swapping the order on these:
https://github.com/krakenjs/beaver-logger/blob/main/src/logger.js#L222-L229

**Comment by westeezy** [comment]
I agree with that. I'll add a ticket to investigate a bit to ensure we want to move forward and make the change in the next version. <3
